cristin-project/src/test/java/no/unit/nva/cristin/projects/create/createSampleProject.feature

```diff
@@ -30,36 +30,55 @@ Feature: API tests for creating sample Cristin Projects
       },
       'contributors': [
         {
-          'type': 'ProjectManager',
           'identity': {
             'type': 'Person',
             'id': 'https://api.dev.nva.aws.unit.no/cristin/person/1684651'
           },
-          'affiliation': {
-            'type': 'Organization',
-            'id': 'https://api.dev.nva.aws.unit.no/cristin/organization/20754.0.0.0'
-          }
+          'roles': [
```

Comment:
Here is the main change. As one can see we now only have one identity per project, but several roles with affiliations per that identity. With the old version we could have duplicate identity causing us to have more contributor entries than people